style.css

```diff
@@ -0,0 +1,143 @@
+@import url('https://fonts.googleapis.com/css2?family=Lexend:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+@import url("https://use.typekit.net/nfq1trc.css");
+
+
+
+* {
+    box-sizing: border-box;
+    padding: 0px;
+    margin: 0px;
+}
+
+body {
+    background-color: #f3f0f2;
+    font-family: 'Lexend', sans-serif;
+    font-weight: 300;
+}
+
+a {
+    color:#24181b;
```

Comment:
try to add another set of rules for the links in the footer. they are currently not visible because the color contrast is so low because it is inheriting this style.